Libplanet.Tests/Tx/TxSuccessTest.cs

```diff
@@ -74,28 +92,46 @@ public TxSuccessTest()
         [Fact]
         public void ConstructorBuilder()
         {
-            Assert.Equal(_blockHash, _fx.BlockHash);
-            Assert.Equal(_txid, _fx.TxId);
-            Assert.Equal(_updatedStates, _fx.UpdatedStates);
-            Assert.Equal(_fungibleAssetsDelta, _fx.FungibleAssetsDelta);
-            Assert.Equal(_updatedFungibleAssets, _fx.UpdatedFungibleAssets);
+            var fxs = new[]
+            {
+                _fx,
+                _fxWithLogs,
+            };
+
+            foreach (var fx in fxs)
+            {
+                Assert.Equal(_blockHash, fx.BlockHash);
+                Assert.Equal(_txid, fx.TxId);
+                Assert.Equal(_updatedStates, fx.UpdatedStates);
+                Assert.Equal(_fungibleAssetsDelta, fx.FungibleAssetsDelta);
+                Assert.Equal(_updatedFungibleAssets, fx.UpdatedFungibleAssets);
```

Comment:
Maybe we need to check for ActionLogList here too?

Reply:
I amended these parts too!